.github/CODEOWNERS

```diff
@@ -39,6 +39,6 @@
 
 # ***** Security-sensitive - always keep these at the end. *****
 
-# CI and CD, Repo admins who can access secrets.
-/.github/workflows/ @brunobowden @crazybob @sammousa
-/.github/CODEOWNERS @brunobowden @crazybob @sammousa
+# CI and CD; Note no secrets shall be stored in this GitHub repo.
```

Comment:
Actually, secrets are going to be stored in this repo -- see #869 where GITHUB_TOKEN will be used.

It's not the same as app store secrets, though, but we might want to update the description here to clarify.

Reply:
We want to avoid privilege escalation by adding me/hunter here.

Do we know what scope of privileges the particular GITHUB_TOKEN has?

Reply:
You can always escalate if you can create branches in this repo. Just create a new one and ci will run for that with the secrets.

Reply:
https://help.github.com/en/actions/configuring-and-managing-workflows/authenticating-with-the-github_token#permissions-for-the-github_token

Reply:
We don't necessarily have to merge #869, security concerns are more important than repo convenience...

Reply:
Actually... I'm not sure it matters whether #869 is merged or not. It [looks like](https://help.github.com/en/actions/configuring-and-managing-workflows/authenticating-with-the-github_token#about-the-github_token-secret) that secret is created automatically anyway